src/rp2_common/hardware_uart/uart.c

```diff
@@ -163,37 +167,42 @@ void uart_set_format(uart_inst_t *uart, uint data_bits, uint stop_bits, uart_par
                    UART_UARTLCR_H_PEN_BITS |
                    UART_UARTLCR_H_EPS_BITS);
 
-    if (was_enabled)
+    if (was_enabled) {
         uart_get_hw(uart)->cr = cr_save;
+    }
 }
 
 void uart_set_fifo_enabled(uart_inst_t *uart, bool enabled) {
     bool was_enabled = uart_is_enabled(uart);
     uint32_t cr_save;
-    if (was_enabled)
+    if (was_enabled) {
         cr_save = uart_disable_before_lcr_write(uart);
-
+    }
     hw_write_masked(&uart_get_hw(uart)->lcr_h,
                    (bool_to_bit(enabled) << UART_UARTLCR_H_FEN_LSB),
                    UART_UARTLCR_H_FEN_BITS);
 
-    if (was_enabled)
+    if (was_enabled) {
         uart_get_hw(uart)->cr = cr_save;
+    }
 }
 
 void uart_set_break(uart_inst_t *uart, bool en) {
     bool was_enabled = uart_is_enabled(uart);
-    uint32_t cr_save;
-    if (was_enabled)
+    uint32_t cr_save = 0;  // stifle warning
```

Comment:
Just out of curiosity, how come this "stifle warning" wasn't needed in the other functions that this PR modifies?

Reply:
I'm not sure!  This one was required, I think the compiler was being more paranoid than it needed to be, no others were.